<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpWindowWindow</class>
 <widget class="QDockWidget" name="HelpWindowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="layout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../res/HelpWindow.qrc">
     <normaloff>:/oxygen/actions/go-home.png</normaloff>:/oxygen/actions/go-home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="statusTip">
    <string>Go to the home page</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../../Core/res/Core.qrc">
     <normaloff>:/oxygen/actions/go-previous.png</normaloff>:/oxygen/actions/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="statusTip">
    <string>Go back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../../Core/res/Core.qrc">
     <normaloff>:/oxygen/actions/go-next.png</normaloff>:/oxygen/actions/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="statusTip">
    <string>Go forward</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../res/HelpWindow.qrc">
     <normaloff>:/oxygen/actions/zoom-in.png</normaloff>:/oxygen/actions/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="statusTip">
    <string>Zoom in the help page contents</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../res/HelpWindow.qrc">
     <normaloff>:/oxygen/actions/zoom-out.png</normaloff>:/oxygen/actions/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="statusTip">
    <string>Zoom out the help page contents</string>
   </property>
  </action>
  <action name="actionNormalSize">
   <property name="icon">
    <iconset resource="../res/HelpWindow.qrc">
     <normaloff>:/oxygen/actions/zoom-original.png</normaloff>:/oxygen/actions/zoom-original.png</iconset>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
   <property name="statusTip">
    <string>Reset the size of the help page contents</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../Core/res/Core.qrc">
     <normaloff>:/oxygen/actions/edit-copy.png</normaloff>:/oxygen/actions/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy the selection to the clipboard</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../res/HelpWindow.qrc">
     <normaloff>:/oxygen/actions/document-print.png</normaloff>:/oxygen/actions/document-print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="statusTip">
    <string>Print the help page contents</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../Core/res/Core.qrc"/>
  <include location="../res/HelpWindow.qrc"/>
 </resources>
 <connections/>
</ui>
